kinto_http/__init__.py

```diff
@@ -644,17 +670,36 @@ def get_records(self, *, collection=None, bucket=None, **kwargs):
                                      collection=collection)
         return self._paginated(endpoint, **kwargs)
 
-    def get_record(self, *, id, collection=None, bucket=None):
-        endpoint = self.get_endpoint('record', id=id,
-                                     bucket=bucket,
-                                     collection=collection)
+    def get_record(self, *, id, collection=None, bucket=None, history_revision=None):
+        if history_revision is not None:
+            endpoint = self.get_endpoint('record_revision',
+                                         id=id,
+                                         bucket=bucket,
+                                         collection=collection,
+                                         history_revision=history_revision
+                                         )
+            logger.info(
+              "Get record with id %r from collection %r in bucket %r by revision %r"
+              % (id, collection or self._collection_name,
+                 bucket or self._bucket_name, history_revision)
+            )
 
-        logger.info(
-          "Get record with id %r from collection %r in bucket %r"
-          % (id, collection or self._collection_name, bucket or self._bucket_name))
+            resp, _ = self.session.request('get', endpoint)
+            return resp.get('data')[0].get('target')
```

Comment:
This will fail (`IndexError`) if the revision is unknown. You should have a test for this ;)

Also, don't use `.get()` if you are sure that the key will be here. Which is the case here because the server should return `["data"]` and `["target"]`